### PR TITLE
Start new version script fixed

### PR DESCRIPTION
**Pull request by olmisa**
* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

A start new version script was fixed. Now script set new version in all needed files.

## Related PRs or issues

[AB#77882](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/77882)

### CODE REVIEW

**Review by annakocheshkova — APPROVED** [review]

**Review by russelarms — DISMISSED** [review]

File: scripts/start-new-version.sh [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
 # Note: Run this from within the root directory
 
 version=$1
 
-./build.sh -s "version.cake" -t "StartNewVersion" -NewVersion="$version"
+./build.sh -s "version.cake" --t="StartNewVersion" --NewVersion="$version"
```
Comment (Jamminroot) [inline scripts/start-new-version.sh]: Is this line tested? That looks like a regular bash named arguments passing, and those are commonly used with single hyphen (as it was previously: `...-t "StartNewVersion"`).

**Review by annakocheshkova — DISMISSED** [review]

File: scripts/start-new-version.sh (hunk 1 above)
Comment (olmisa) [reply scripts/start-new-version.sh]: Yes, it was tested.

Comment (annakocheshkova) [reply scripts/start-new-version.sh]: ~I use `./build.ps1 -target Externals` format...~ Nevermind it's shell

Comment (olmisa) [reply scripts/start-new-version.sh]: Reworked

**Review by guperrot — APPROVED** [review]

**Review by Jamminroot — APPROVED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**